File: WebApp/Models/Auth/LoginDTO.cs
namespace WebApp.Models.Auth
{
    using System.ComponentModel.DataAnnotations;

	public class LoginDTO
	{
		[Required]
		public string Email { get; set; }

		[Required]
		[DataType(DataType.Password)]
		public string Password { get; set; }
	}
}

File: Frontend/src/models/auth/LoginDTO.ts
export class LoginDTO {
    public email: string;
    public password: string;

    constructor(dto: LoginDTO = null) {
        if (dto) {
            this.email = dto.email;
            this.password = dto.password;
        }
        else {
            this.email = "";
            this.password = "";
        }
    }
}